ouroboros-consensus-diffusion/src/ouroboros-consensus-diffusion/Ouroboros/Consensus/Node.hs

```diff
@@ -668,6 +720,13 @@ mkNodeKernelArgs
       , miniProtocolParameters  = defaultMiniProtocolParameters
       , blockFetchConfiguration = defaultBlockFetchConfiguration
       , keepAliveRng
+      , gsmArgs = GsmNodeKernelArgs {
+          gsmAntiThunderingHerd  = snd (split keepAliveRng)
```

Comment:
Maybe `keepAliveRng` should be renamed to sound more "generic" and then splitted for different domain specific purposes?

Reply:
I pushed the split all the way up to a new `llrnGsmAntiThunderingHerdRng` field 👍